src/grammar.lua

```diff
@@ -453,3 +456,17 @@ function pmatch(patt)
     end
   end)
 end
+
+_M.anyOf	= anyOf
```

Comment:
Instead of setting fields in the end, change the definition of `function anyOf(...)` to `function grammar.anyOf(...)`.

(Note that with the approach in this diff, `anyOf` remains a global function, which is not desired.)